setup.cfg

```diff
@@ -1,6 +1,6 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=test_settings
-addopts = --verbose --nomigrations --reuse-db -s --ignore ve --flake8 --cov --nomigrations
+addopts = --verbose --nomigrations --reuse-db -s --ignore ve --flake8
```

Comment:
ofc it's a bit hard to test well. But 2 runs w/o `--cov` took ~70s on my machine. And 2 runs with `--cov` took ~100s. So I'd say it does give a noticeable speedup.